tests/helpers/main.ts

```diff
@@ -91,13 +91,19 @@ export const zipFixture = async function (
 
 export const zipCheckFunctions = async function (
   fixture: string[] | string,
-  { length = 1, fixtureDir = FIXTURES_DIR, tmpDir, opts }: ZipOptions & { tmpDir: string },
+  { length = 1, fixtureDir = FIXTURES_DIR, tmpDir, opts = {} }: ZipOptions & { tmpDir: string },
 ): Promise<ZipReturn> {
   const srcFolders = Array.isArray(fixture)
     ? fixture.map((srcFolder) => `${fixtureDir}/${srcFolder}`)
     : `${fixtureDir}/${fixture}`
 
-  const files = await zipFunctions(srcFolders, tmpDir, opts)
+  let basePath
+
+  if (!opts.basePath && typeof fixture === 'string') {
+    basePath = resolve(fixtureDir, fixture)
```

Comment:
All tests should have a `basePath` property, otherwise we'll look for modules and `tsconfig.json` files that belong to the parent zip-it-and-ship-it module, not the fixture.

The proper way would be to add it to each test individually, but I'm adding this as a fallback temporarily and will follow up in a separate PR.

This is also why you'll see some changes around Node modules in the fixtures. We were relying on modules from zip-it-and-ship-it in some tests, which we shouldn't.